runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,46 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: vim.tohtml                                            *vim.tohtml*
+
+tohtml({winid}, {outfile}, {opt})                   *require"tohtml".tohtml()*
```

Comment:
This is not a proper tag. Can we do `vim.tohtml.tohtml()`? Also, `vim.tohtml` or `tohtml`?

Reply:
tohtml is more of an external plugin so `tohtml` instead of `vim.tohtml`.